typingdna_activelock/README.md

```diff
@@ -0,0 +1,55 @@
+
+
+# TypingDNA ActiveLock
+
+## Overview
+
+[TypingDNA ActiveLock][3] is a Continuous Endpoint Authentication app that helps prevent unauthorized access to your company computers. Once installed on a user's PC, it continuously verifies the user by the way they type. If an unauthorized typing pattern is detected, ActiveLock can instantly lock the computer and log the data to your desired logging platform (such as Datadog).
+
+To visualize your data in Datadog, a custom ActiveLock app needs to be configured and installed. This is the same install for all of your company computers.
+
+
+## Setup
+
+### Configuration
+
+To generate a Datadog API key:
+
+1. Navigate to [Organization settings > API keys][4] in your Datadog account.
+2. Click **+ New Key** to generate an API key.
+
+To get your custom install app:
+
+1. Complete [this custom install form][7] by submitting your newly generated API key and Datadog site region (such as US1 or EU), along with your company details.
+2. You will then receive a custom ActiveLock app that you can install on your company computers and more information over email.
+3. After installation, your ActiveLock logs should start to appear in [Log Explorer][5].
```

Comment:
Just a reminder to update the instructions once you have built out the page and form for customers to submit their api key.

Reply:
Thanks, I will, currently waiting for the page to be published, may take a few more days.